validation/api/api.lock.json

```diff
@@ -931,13 +931,18 @@
   {
     "name": "Random",
     "exports": [
-      "public class AsyncRandom(",
+      "public class AsyncRandom(state : State, generator : () -> async* Blob)",
       "public let blob : shared () -> async Blob",
       "public func crypto() : AsyncRandom",
-      "public func fast(seed : Nat64) : Random",
-      "public func fromAsyncGenerator(generator : () -> async* Blob) : AsyncRandom",
-      "public func fromGenerator(generator : () -> Blob) : Random",
-      "public class Random("
+      "public func cryptoFromState(state : State) : AsyncRandom",
+      "public func cryptoState() : State",
+      "public func emptyState() : State",
```

Comment:
When to use which of the two? I'd just remove one of them, maybe `cryptoState` because it is just equivalent to `emptyState`